packages/next/next-server/server/spr-cache.ts

```diff
@@ -155,6 +161,8 @@ export async function setSprCache(
     revalidateAfter: calculateRevalidate(pathname),
   })
 
+  // TODO: This option needs to cease to exist unless it stops mutating the
+  // `next build` output's manifest.
```

Comment:
We don't mutate the prerender manifest on the disk currently, it was only discussed but not added afaik

Reply:
But it's mutating the prerendered `.html` files, right?

Reply:
Yeah which I believe we want to allow debugging but it might make sense to disable by default. We can discuss this part further separate since this PR doesn't affect it 👍